src/Uno.UWP/Graphics/Display/DisplayInformation.wasm.cs

```diff
@@ -152,6 +161,24 @@ private static DisplayOrientations ParseJsOrientation(string jsOrientation)
 				_ => DisplayOrientations.None,
 			};
 		}
+
+		private static string ToJsOrientation(DisplayOrientations orientations)
+		{
+			return orientations switch
+			{
+				DisplayOrientations.Landscape => "landscape-primary",
+				DisplayOrientations.LandscapeFlipped => "landscape-secondary",
+				DisplayOrientations.Portrait => "portrait-primary",
+				DisplayOrientations.PortraitFlipped => "portrait-secondary",
+				_ => "any"
+			};
```

Comment:
`DisplayOrientations` is a flags enum. So, multiple values can be specified.

I think we should well understand the meaning of them, and use the corresponding JS orientation.

Reply:
It seems like if multiple values are specified, the system gets to choose which orientation.

So `Portrait | PortraitFlipped` can be mapped to the JS `portrait`, while `Landscape | LanscapeFlipped` can be mapped to the JS `landscape`.

However, for other values, I don't think there's a suitable mapping. A solution might be passing a list to the TypeScript function, and then return when setting to one of the values succeed?

Reply:
@MartinZikmund Could you advise here?

Reply:
I've removed the mentioned code. Instead, I'm passing the value as-is to the JavaScript code (passing an int is much cheaper than passing an array of strings, which would require JSON serialization).

The JavaScript code would then decode the enum to a list of `OrientationLockType`, and attempt to lock the orientation at one of the types specified.

Reply:
This code is outdated, so I'll mark this as resolved.